src/submission/validation.ts

```diff
@@ -8,7 +8,7 @@ import {
   RegistrationToCreateRegistrationFieldsMap,
```

Comment:
:eyes: I don't know how this file still exists? I remember it was moved to `./validation-clinical`...

Reply:
:ghost:  dunno will delete it again